data/translations/nl.ts

```diff
@@ -0,0 +1,417 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="nl_NL">
+<context>
+    <name>CustomEditor</name>
+    <message>
+        <location filename="../../src/customeditor/customeditor.cpp" line="32"/>
+        <source>RazerGenie - Custom Editor</source>
+        <translation>RazerGenie - Aangepaste bewerker</translation>
```

Comment:
I guess this translation is accurate, but it sounds really strange. I don't know how to improve it but I wouldn't personally use this.

Reply:
"bewerker" is used in lots of Dutch translations of applications, so it's correct and consistent.